Source: https://github.com/epoupon/fileshelter/pull/111.patch
--
From 82bd2b7fcf81b0d13d9304ebbd5a0d4cb0332be5 Mon Sep 17 00:00:00 2001
From: Valerio Baldisserotto <svalo@users.noreply.github.com>
Date: Thu, 30 May 2024 14:55:13 +0200
Subject: [PATCH] Adds missing algorithm include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Before this change compilation failed with:

```cmake
src/fileshelter/ui/ShareCreateFormView.cpp:285:29: error: ‘count_if’ is not a member of ‘std’
  285 |                 return std::count_if(std::cbegin(files), std::cend(files),
```
---
 src/fileshelter/ui/ShareCreateFormView.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/fileshelter/ui/ShareCreateFormView.cpp b/src/fileshelter/ui/ShareCreateFormView.cpp
index ff9b79a..0dc3007 100644
--- a/src/fileshelter/ui/ShareCreateFormView.cpp
+++ b/src/fileshelter/ui/ShareCreateFormView.cpp
@@ -19,6 +19,7 @@
 
 #include "ShareCreateFormView.hpp"
 
+#include <algorithm>
 #include <numeric>
 
 #include <Wt/WApplication.h>
